its/src/test/java/com/sonar/it/scanner/msbuild/utils/BuildCommand.java

```diff
@@ -69,9 +69,10 @@ public BuildCommand addArgument(String... arguments) {
   public BuildResult execute() {
     var command = createCommand();
     var result = new BuildResult();
-    LOG.info("Command line: {}", command.toCommandLine());
+    LOG.info("Build command start: '{}' in {}", command.toCommandLine(), command.getDirectory());
```

Comment:
Without the executing directory, most commands can not be tracked down to the test method that was executing them.